block/stream/output/schedule_service.proto

```diff
@@ -64,6 +64,17 @@ import "basic_types.proto";
  * the original transaction.
  */
 message CreateScheduleOutput {
+    /**
+     * A schedule identifier.
+     * <p>
+     * If the status of the transaction is `SUCCESS`, this value SHALL be
+     * set to the identifier of the schedule that was created. If the
+     * transaction status is `IDENTICAL_SCHEDULE_ALREADY_CREATED`, this
+     * value SHALL be set to the identifier of the existing schedule that
+     * is identical to the one that was attempted to be created. For any
+     * other status, this value SHALL NOT be set.
+     */
+    proto.ScheduleID schedule_id = 1;
```

Comment:
This field is unnecessary, and was expressly removed from the block stream HIP.
The schedule that is created will be present in the State Changes, including the ID.

Reply:
We discussed this before @jsync-swirlds , as per the comment there isn't a corresponding `StateChanges` item when the `ScheduleCreate` result is `IDENTICAL_SCHEDULE_ALREADY_CREATED`.

Reply:
I had forgotten.  Thanks for the reminder.